### DISCUSSION TITLE
DHCP does not give the good IP and the Web service only listening on Ipv6

### DISCUSSION THREAD

**Discussion by needsone** [cobbler/cobbler#3352] [General]
I did finish the Quickstart but I have some issues.
The dhcp server give Ips but not the one I would like and the tftpd does not give any content.

```
cobbler system edit --name=mydebian --interface=ens192 --mac="00:50:56:90:42:41" --ip-address=10.10.20.201 --netmask=255.255.255.0 --static=1 --dns-name=mcp.cyfer.io
```

I don't use IPv6 fpr cobbler, the dhcp IPv6 is off, but the apache2 is listening on IPv6.

```
[root@cobbler install]# netstat -nlp | grep 80
tcp6       0      0 :::80                   :::*                    LISTEN      1043/httpd          
```

Here is a zip with my settings.yaml and dhcp.template

[settings-yaml.zip](https://github.com/cobbler/cobbler/files/10756884/settings-yaml.zip)

**Comment by SchoolGuy** [comment]
@needsone So multiple remarks on first sight:

- port 80 is usually the Apache webserver
- port 69 is tftp

Do you have other DHCP servers in your network? Because normally you only want to have a single DHCP server in your network. This at least would explain why your device got another IP address.

If the TFTP server is running you can try out another client that is already installed on the network and try to fetch files. A common issue is that people try out it locally on the same server. The problem, in the end, is a firewall that allows access via localhost but not on a remote interface.
